examples/typescript/apply/apply-example.ts

```diff
@@ -31,7 +31,9 @@ export async function apply(specPath: string): Promise<k8s.KubernetesObject[]> {
             // block.
             await client.read(spec);
             // we got the resource, so it exists, so patch it
-            const response = await client.patch(spec);
+            const options = { headers: { 'content-type': 'application/merge-patch+json' } }
```

Comment:
Like `kubectl apply`, the object API defaults to strategic merge patching, https://github.com/kubernetes-client/javascript/blob/b218eb9e7f6080239b7061d42a088749b1d4a749/src/object.ts#L444-L446 . Without an explanation or comment, it seems to me changing from the default may lead to confusion.